### DISCUSSION TITLE
Problems with face generation in Fooocus 2.5

### DISCUSSION THREAD

**Discussion by Grisha3D** [lllyasviel/Fooocus#3312] [Q&A] [answered]
I encountered a dramatic deterioration in the generation of faces in Fooocus 2.5; such problems were not observed in Fooocus 2.4. Has anyone encountered problems with the quality of generation in  Fooocus 2.5?
![2024-07-19_05-31-13_3886](https://github.com/user-attachments/assets/0aa828b7-9867-4e7b-8b39-7f4f70cba106)
![2024-07-19_05-04-09_5431](https://github.com/user-attachments/assets/a5c4df21-105d-40d6-a6e3-e1dcf3ff500a)

**Comment by mashb1t** [comment, marked as answer]
If you use the realistic preset you can always jump back from the 1.0 to the old 0.4 beta LoRA. If you don't use the realistic or anime preset nithing has changed in terms of model updates / handling in the code.

**Comment by Grisha3D** [comment]
I don't use presets, I run run.bat. There is a problem with faces and a colab (https://colab.research.google.com/github/lllyasviel/Fooocus/blob/main/fooocus_colab.ipynb) version of fooocus. I downloaded Fooocus Control, there is no such problem...

![2024-07-19_14-13-27_6220](https://github.com/user-attachments/assets/f53c37a1-ac1a-4fb9-a15c-70a554cd3675)

**Reply by mashb1t** [reply]
Make sure to use the same model and seed when making comparisons between Fooocus forks, because if you don't do this they're not indicative of individual changes in the code.
